Disable annoying warning for newer GTK versions

https://github.com/wmww/gtk-layer-shell/blob/master/compatibility.md
> GTK Layer Shell has a hard time explicitly supporting each version of
> GTK, but *new unsupported versions almost always work in practice*.

--- a/gtk-priv/h/common.h
+++ b/gtk-priv/h/common.h
@@ -24,7 +24,7 @@
 }
 
 void gtk_priv_warn_gtk_version_may_be_unsupported() {
-    static gboolean shown = FALSE;
+    static gboolean shown = TRUE;
     if (shown)
         return;
     g_warning(
